pkg/jobrunaggregator/jobrunaggregatorlib/disruption.go

```diff
@@ -48,6 +49,44 @@ type BackendDisruption struct {
 	DisruptionMessages []string
 }
 
+// defined in origin/platformidentification/types.go
+// minimized here
+type ClusterData struct {
+	MasterNodesUpdated string
```

Comment:
Wondering about a bool here, but the Y/N string does give us tri-state: yes, no, we have no idea.

Reply:
Yes so all the prior data will be empty (don't want to force it to false or true) and in the case that we can't determine the default is empty since we have no idea as you mentioned.